packages/veritone-react-common/src/components/LogoDetectionEngineOutput/index.js

```diff
@@ -0,0 +1,198 @@
+	};
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			isExpanded: false
+		};
+
+		this.onEngineChanged = this.onEngineChanged.bind(this);
+		this.onExpandViewClicked = this.onExpandViewClicked.bind(this);
+	}
+
+	//================================================================================
+	//=============================Interaction Functions==============================
+	//================================================================================
+	onItemSelected = item => (event) => {
+		let itemInfo = {
+			item: item,
+			sourceEngineId: this.state.sourceEngineId || this.props.data[0].sourceEngineId
+		}
+
+		if (this.props.onItemSelected) this.props.onItemSelected(itemInfo);
+	}
+
+	onEngineChanged(event) {
+		let selectedID = event.target.value;
+		this.setState({
+			sourceEngineId: selectedID
+		});
+
+		if (this.props.onEngineSelected) this.props.onEngineSelected(selectedID);
+	}
+
+	onExpandViewClicked(event) {
+		let newStatus = !this.state.isExpanded;
+		this.setState({
+			isExpanded: newStatus
+		});
+
+		if (this.props.onExpandViewClicked) this.props.onExpandViewClicked(newStatus);
+	}
+
+	//================================================================================
+	//================================Helper Functions================================
+	//================================================================================
+	/**
+	 * draw engine dropdown selections
+	 * @param {Array<{sourceEngineId:string, sourceEngineName:string, taskId: string, series:Array>} data - engine output data
+	 * @return MUI Select Element
+	 */
+	drawEngineSelection(data) {
+		if (!data || data.length === 0) return '';
+
+		let selectedID = this.state.sourceEngineId || data[0].sourceEngineId;
+		return (
+			<Select value={selectedID} onChange={this.onEngineChanged}>
+				{
+					data.map((engineInfo) => {
+						return (
+							<MenuItem
+								value={engineInfo.sourceEngineId}
+								key={'engine-logo-' + engineInfo.sourceEngineId}
+							>
+								{engineInfo.sourceEngineName}
+							</MenuItem>
+						);
+					})
+				}
+			</Select>
+		);
+	}
+
+	/**
+	 * Draw detected logo entry in the series
+	 * @param {Array<{sourceEngineId:string, series:Array<{startTimeMs:number, endTimeMs:number, logo:{label:string}}>>} data - engine output data
+	 * @return {Array<PillButton>}
+	 */
+	drawItems(data) {
+		let currentPlayTime = this.props.mediaPlayerTime;
+		let itemClassName = this.props.itemClassName;
+		let itemInfoClassName = this.props.itemInfoClassName;
+		let itemLabelClassName = this.props.itemLabelClassName;
+		let selectedEngineID = this.state.sourceEngineId || data[0].sourceEngineId;
+
+		let items = [];
+		if (data && data.length > 0) {
+			data.map((engineInfo) => {
+				if (engineInfo.sourceEngineId === selectedEngineID) {									//Look for selected engine
+
+					// Sort detected logos by there start time and end time
+					engineInfo.series.sort(function (item1, item2) {
```

Comment:
To me, it's much easier to read
```
import {sortBy} from lodash;
engineInfo.series = sortBy(engineInfo.series, 'startTimeMs', 'endTimeMs');
```
People gots all sorts of opinions on lodash vs. vanilla JS so I'm fine to defer to the consensus on this one but I do see lodash in package.json so I think it's fine to use here.